build.sbt

```diff
@@ -86,9 +86,9 @@ ThisBuild / semanticdbVersion := scalafixSemanticdb.revision
 
 // NOTE: Include these settings in any projects that require Apalache's TLA+ modules
 lazy val tlaModuleTestSettings = Seq(
-    // we have to tell SANY the location of Apalache modules for the tests
-    Test / fork := true, // Forking is required for the system options to take effect in the tests
-    Test / javaOptions += s"""-DTLA-Library=${(ThisBuild / baseDirectory).value / "src" / "tla"}""",
+    // Forking is required for the system options to take effect in the tests
+    // It also ensures that tests run from their respective sub-project directories
+    Test / fork := true
 )
```

Comment:
Do we still need the forking if we don't need set this option any more? What is the importance of running from the sub-project directories?

Regardless, I think the comment about the "system options" can be removed, since those are not being set anymore?

Reply:
There's two issues here, paths and failing tests due to parallelism; I've opened https://github.com/informalsystems/apalache/issues/1577 to follow up on this.
The comment's fixed in e56c96e9